### PR TITLE
Work/bboyer/user tests

### PR DESCRIPTION
**Pull request by BlBoyer**

### CODE REVIEW

File: src/CVPZ.Application/Common/Behaviors/UserTracker.cs [hunk 1]
```diff
@@ -20,7 +20,6 @@ public UserTracker(IHttpContextAccessor httpContextAccessor, IMediator mediator,
         this._mediator = mediator;
         _logger = logger;
     }
-
```
Comment (CarBar) [inline src/CVPZ.Application/Common/Behaviors/UserTracker.cs]: I like empty lines in between curly braces of top level things.

**Review by CarBar — CHANGES_REQUESTED** [review]
Great job. One minor line I would like in place.

File: src/CVPZ.Application/Common/Behaviors/UserTracker.cs (hunk 1 above)
Comment (BlBoyer) [reply src/CVPZ.Application/Common/Behaviors/UserTracker.cs]: resolved :)

Comment (BlBoyer) [reply src/CVPZ.Application/Common/Behaviors/UserTracker.cs]: resolved :)

**Comment by CarBar** [comment]
Thanks for contributing @BlBoyer

**Review by CarBar — APPROVED** [review]

**Pull request merged**